### PR TITLE
Add Virtio support to CIV launch scripts

### PR DESCRIPTION
**Pull request by renchenglei**
This helps to launch virtio with scripts

Tracked-On: OAM-91381
Signed-off-by: Ren Chenglei <chenglei.ren@intel.com>

### CODE REVIEW

File: groups/device-specific/caas/start_android_qcow2.sh [hunk 1]
```diff
@@ -471,6 +500,8 @@ if [[ "$vno" > "5.0.0" ]]; then
 	create_snd_dummy
 	if [[ $1 == "--gvtd" ]]; then
 		setup_gvtd
+	elif [[ $1 == "--virtio" ]]; then
```
Comment (YadongQi) [inline groups/device-specific/caas/start_android_qcow2.sh:503]: rename "--virtio" with "--virtio-gpu". "--virtio" may confuse user.

File: groups/device-specific/caas/start_android_qcow2.sh [hunk 2]
```diff
@@ -433,6 +439,29 @@ function launch_hwrender_gvtd(){
 	cleanup_vsock_host_utilities
 }
 
+function launch_virtio(){
```
Comment (YadongQi) [inline groups/device-specific/caas/start_android_qcow2.sh:442]: rename launch_virtio() to launch_virtio_gpu() or launch_virtio_render().

File: groups/device-specific/caas/start_android_qcow2.sh [hunk 3]
```diff
@@ -51,6 +51,12 @@ function setup_gvtd(){
 	return $res
 }
 
+function setup_virtio(){
```
Comment (YadongQi) [inline groups/device-specific/caas/start_android_qcow2.sh:54]: rename to setup_virtio() to setup_virtio_gpu().

**Comment by renchenglei** [comment]
Thanks @YadongQi , I have updated the patch, please help take a review, :)

File: groups/device-specific/caas/start_android_qcow2.sh [hunk 4]
```diff
@@ -481,6 +512,8 @@ if [[ "$vno" > "5.0.0" ]]; then
 	if [[ $? == 0 ]]; then
 		if [[ $1 == "--gvtd" ]]; then
 			launch_hwrender_gvtd
+		elif [[ $1 == "--virtio" ]]; then
```
Comment (YadongQi) [inline groups/device-specific/caas/start_android_qcow2.sh:515]: --virtio_gpu

File: groups/device-specific/caas/start_android_qcow2.sh [hunk 5]
```diff
@@ -481,6 +512,8 @@ if [[ "$vno" > "5.0.0" ]]; then
...
+			launch_virtio
```
Comment (YadongQi) [inline groups/device-specific/caas/start_android_qcow2.sh:516]: launch_virtio_gpu

**Comment by renchenglei** [comment]
Thanks @YadongQi for the reminding, it's my fault, and missed some changes.

**Comment by renchenglei** [comment]
Thanks @YadongQi, and done! :)

**Comment by renchenglei** [comment]
@YadongQi, could you help take a review. I rebased the code, and add the support for sw render.

**Comment by renchenglei** [comment]
Previously, I met resolution issues with qxl-vga, so I changed to bochs-display. I just tried, and qxl-vga also works fine with 480 * 360 resolution. If there is any concern, I could revert back to qxl-vga.

**Comment by YadongQi** [comment]
So the resolution could not higher than 480*360 for swrender?  Is this a known issue to CiV?

I just want to know the differences between qxl-vga and bochs-display. Could you evaluate the stability/performance of them and then choose one suite for CiV? Thanks.

**Comment by renchenglei** [comment]
As kernel has upgraded to 5.4 and qemu parameter has changed(e.g fullscreen), when the resolution is higher than 480 * 360, only part of Android is shown on screen. And it seems that no one use sw render, and we haven't updated sw render scripts. 
It is not worth to waste too much time to do stability/performance test for sw render, if there is concern, I could change back to qxl-vg.

**Comment by YadongQi** [comment]
OK, then please change back to qxl-vga.

**Comment by renchenglei** [comment]
Ok, thanks and done! :)

**Comment by renchenglei** [comment]
@sgnanase, could you help take a review of this changes, thanks a lot!

**Pull request merged**